src/Schema/Events/Search.ts

```diff
@@ -0,0 +1,89 @@
+import { ActionType } from "."
+import { ContextModule } from "../Values/ContextModule"
+import { OwnerType, PageOwnerType } from "../Values/OwnerType"
+
+/**
+ * Schemas describing Search events
+ * @packageDocumentation
+ */
+
+/**
+ * A user focuses on a search box
+ *
+ * This schema describes events sent to Segment from [[FocusedOnSearchInput]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "focusedOnSearchInput",
+ *    context_module: "header",
+ *    context_page_owner_type: "home"
+ *  }
+ * ```
+ */
+export interface FocusedOnSearchInput {
+  action: ActionType.focusedOnSearchInput
+  context_module: ContextModule
+  context_page_owner_type: PageOwnerType
+  context_page_owner_id?: string
+  context_page_owner_slug?: string
+}
+
+/**
+ * A user selects an item from search
+ *
+ * This schema describes events sent to Segment from [[SelectedItemFromSearch]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "selectedItemFromSearch",
+ *    context_module: "priceEstimate",
+ *    context_page_owner_type: "consign",
+ *    owner_type: "artist",
+ *    owner_id: "5df3e3fa485efe0012c37055",
+ *    owner_slug: "andy-warhol",
+ *    query: "andy warhol"
+ *  }
+ * ```
+ */
+export interface SelectedItemFromSearch {
+  action: ActionType.selectedItemFromSearch
+  context_module: ContextModule
+  context_page_owner_type: PageOwnerType
+  context_page_owner_id?: string
+  context_page_owner_slug?: string
+  destination_page_owner_type?: PageOwnerType
+  destination_page_owner_id?: string
+  destination_page_owner_slug?: string
+  owner_type: OwnerType
```

Comment:
what is the owner type that is neither context nor destination?

Reply:
This is specific to the [new consign page](https://www.figma.com/file/I7a7Iy3yCFi7Pj1VguqMFG/Sell-Art-Landing-%5BWeb%5D?node-id=1155%3A0). In the new design, you can search for an artist to view their market insights. It does not take you to a new page; it just displays the result on your existing page, so I added this additional owner info so that we can track what users are choosing.

Reply:
That's why the destination_page_owner_type field is not required!